readme.md

```diff
@@ -72,6 +72,15 @@ var value = currency("123.45");
 currency(value);    // 123.45
 ```
 
+Currency is in dollars by default but can be parsed as cents. This will respect the precision option when parsing.
```

Comment:
```suggestion
Currency accepts decimal values (i.e. `1.23`) with a default precision of 2, but can accept a minor currency unit (e.g. cents in a dollar). This will respect the precision option when parsing.
```